cmd/helm/upgrade.go

```diff
@@ -286,8 +285,6 @@ func (u *upgradeCmd) run() error {
 				force:        u.force,
 				timeout:      u.timeout,
 				wait:         u.wait,
-				description:  "",
-				revision:     releaseHistory.Releases[0].Version,
```

Comment:
This needs to be here to roll back to the previous version, correct?

Reply:
If I recall, by omitting the revision number, the last version number will be selected. I'll have to double-check this as I thought it'd fetch the latest deployed release rather than current release - 1

Basically I was trying to protect us from the following scenario:

```
1 -> DEPLOYED
2 -> FAILED
```

If we call `helm upgrade --atomic` in this case and it fails a second time, the assumption here is that it may roll back to version 2. I'm pretty sure we want to roll back to version 1 in this case.

Reply:
Yeah, as long as it fetches the last deployed release, I think it should be good